backend/src/main/java/de/neuefische/backend/model/RandMCharacterWithNamePrefix.java

```diff
@@ -0,0 +1,28 @@
+package de.neuefische.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class RandMCharacterWithNamePrefix extends RandMCharacter {
+    @SuppressWarnings("squid:S116")
```

Comment:
why do you want to suppress compiler warnings at this point?

Reply:
warning was for naming convention for java, which was a problem since our filter method for mongoDB has another convention. the only problem with that line was the _ due to java convention we shouldnt use it for naming references. 
but mongo uses it so we had to comply with mongo here